_data/devsite_redirects.yaml

```diff
@@ -0,0 +1,46 @@
+---
+gcloud: gcloud
+google-cloud: google-cloud
+google-cloud-asset: cloudasset
+google-cloud-bigquery: bigquery
```

Comment:
Why are the docs not using their gem names? This seems confusing to me.

Reply:
Docuploader uses the name field of repo-metadata, which is supposed to be the dns name of the API service. 

I have mixed feelings about this.

I agree that it's more confusing if you arrive there from the library repo. But I think it's probably less confusing if you arrive there from a product page. If all languages follow spec, it would also be less confusing for a user to bounce between languages.

Reply:
Reverted back to using gem name